olive/model/handler/mixin/dummy_inputs.py

```diff
@@ -15,16 +15,11 @@ class DummyInputsMixin:
     the dummy data is used to evaluate the latency if user doesn't provide the data for evaluation.
     """
 
-    def _get_dummy_inputs_from_io_config(self, force_kv_cache: bool = False, filter_hook=None, filter_hook_kwargs=None):
+    def _get_dummy_inputs_from_io_config(self, filter_hook=None, filter_hook_kwargs=None):
```

Comment:
please undo this change. this argument is used by the hf model handler https://github.com/microsoft/Olive/blob/0786cd091a745cc7eb5f1ede22527d2f235fff99/olive/model/handler/hf.py#L116

Reply:
synced offline